src/main/java/list.java

```diff
@@ -0,0 +1,42 @@
+public class List {
+    private static String[] currentList = new String[100];
+    public static int listSize = 0;
+    private static int[] isDone = new int[100];
+
+    public static void addToList(String input) {
+        currentList[listSize] =  input;
+        listSize++;
+        System.out.println("____________________________________________________________\n"
+                + "Added: " + input + "\n"
+                + "____________________________________________________________\n");
+    }
+
+    public static void printItems() {
+        if (listSize == 0) {
+            System.out.println("The list is empty.");
```

Comment:
The if and else makes the happy path not prominent. 
```suggestion
            System.out.println("The list is empty.");
            return;
```